dist/virtual-keys.js

```diff
@@ -93,11 +108,27 @@ class VirtualKeysPanel extends LitElement {
       type: 'virtual_keys/create_token',
       name: this.name,
       user_id: this.user,
-      minutes: parseInt(this.expire, 10),
+      minutes: this.expire ? parseInt(this.expire, 10) : differenceInMinutes(this.expirationDate)
```

Comment:
yeah, I like the idea of having the Date selector for expiration and doing this on the client side instead of passing the date to the server